docs/polkadot/implementors-guide/guide.md

```diff
@@ -0,0 +1,1013 @@
+                    /                                  \
+                    |            Runtime               |
+                    |                                  |
+                    \_________(Runtime API )___________/
+                                |       ^
+                                V       |
+               +----------------------------------------------+
+               |                                              |
+               |                   Node                       |
+               |                                              |
+               |                                              |
+               +----------------------------------------------+
+                                   +  +
+                                   |  |
+               --------------------+  +------------------------
+                                 Transport
+               ------------------------------------------------
+
+```
+
+
+It is also helpful to divide Node-side behavior into two further categories: Networking and Core. Networking behaviors relate to how information is distributed between nodes. Core behaviors relate to internal work that a specific node does. These two categories of behavior often interact, but can be heavily abstracted from each other. Core behaviors care that information is distributed and received, but not the internal details of how distribution and receipt function. Networking behaviors act on requests for distribution or fetching of information, but are not concerned with how the information is used afterwards. This allows us to create clean boundaries between Core and Networking activities, improving the modularity of the code.
+
+```
+          ___________________                    ____________________
+         /       Core        \                  /     Networking     \
+         |                   |  Send "Hello"    |                    |
+         |                   |-  to "foo"   --->|                    |
+         |                   |                  |                    |
+         |                   |                  |                    |
+         |                   |                  |                    |
+         |                   |    Got "World"   |                    |
+         |                   |<--  from "bar" --|                    |
+         |                   |                  |                    |
+         \___________________/                  \____________________/
+                                                   ______| |______
+                                                   ___Transport___
+
+```                                                   
+
+
+Node-side behavior is split up into various subsystems. Subsystems are long-lived workers that perform a particular category of work. Subsystems can communicate with each other, and do so via an Overseer that prevents race conditions.
```

Comment:
> that prevents race conditions

For completeness sake, would you mind describing how the serialization point *Overseer* helps prevent race conditions?

Reply:
It's currently described in the overseer section below